
std_msgs/Header header

## 1任务开始
bool flag_track_on

## 任务类型 1定位 2本机追踪 3TBD
int32 track_mission_type

## 相机位置 1向下 2向前TODO 3TBD
int32 camera_type

## 协作者编号
uint8 collaborator_num


